code/modules/bitrunning/components/service_style.dm

```diff
@@ -0,0 +1,30 @@
+/// Gives the appearance of being an agent
+/datum/element/service_style
```

Comment:
This should be in the elements folder instead of in a module

Reply:
You're gonna hate it here if you look around